docs/elasticsearch/snapshot-restore.md

```diff
@@ -333,13 +333,13 @@ POST _snapshot/my-repository/2/_restore
 
 Setting | Description
 :--- | :---
-indices | The indices that you want to restore. You can use `,` to create a list of indices, `*` to specify an index pattern, and `-` to exclude certain indices. Don't put spaces between items. Default is all indices.
+indices | The indices you want to restore. You can use `,` to create a list of indices, `*` to specify an index pattern, and `-` to exclude certain indices. Don't put spaces between items. Default is all indices.
 ignore_unavailable | If an index from the `indices` list doesn't exist, whether to ignore it rather than fail the restore operation. Default is false.
 include_global_state | Whether to restore the cluster state. Default is false.
 include_aliases | Whether to restore aliases alongside their associated indices. Default is true.
 partial | Whether to allow the restoration of partial snapshots. Default is false.
 rename_pattern | If you want to rename indices as you restore them, use this option to specify a regular expression that matches all indices you want to restore. Use capture groups (`()`) to reuse portions of the index name.
-rename_replacement | If you want to rename indices as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching index name, `$1` to include the content of the first capture group, etc.
+rename_replacement | If you want to rename indices as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching index name, `$1` to include the content of the first capture group, and so on.
```

Comment:
I don't see the need to avoid the small handful of popular Latinisms: etc., i.e., e.g.

Reply:
Ack. My brain is still automatically adhering to the style guide at my previous company :)